### ISSUE TITLE
Feature request: Add the option to set id in elements when exporting as SVG

### ISSUE THREAD

**Issue by abusedmedia** [excalidraw/excalidraw#2744]
It'd be great to have this option, pretty much like Figma/Illustrator/etc do actually (exploiting the element name, usually).
The DOM of the SVG can be exploited much more easily by site/app that host them.
Is it something you might consider?

**Comment by dwelle** [comment]
Do you mean adding element types to the svg as an `id`, e.g. `Reactangle1`, `Rectangle2`, `Ellipse1` etc? This could be done quite easily, if we decide to.

If on the other hand you want a UI for people to name their elements manually, then that's probably not something we'll do (although we could put it inside "Stats for nerds" if people really wanted this).

**Comment by abusedmedia** [comment]
Hi @dwelle 

That's great! Adding the automatic `id`, as you suggested, is already a very good compromise.

I understand the implication of additional UI elements, thus, I'd go with an always present property.

Honestly, I don't see issues about the size, I believe that the size of an `id="Rectangle1"` chunk is pretty forgettable if compared to the average size of a tipical rough element.

With this little add-on, an excalidraw document can become dynamic and interactive much more easily.

**Comment by jugglingcats** [comment]
I agree this would be a useful feature. The ids in Exalidraw are more like guids than "friendly" names `Rectangle1` and so on. Example below from clipboard.

```
{
  "id":"TnYBcnsIpQ3jcOB-yT8kR",
  "type":"rectangle",
  "x":280,
  "y":120,
  ...
}
```
It would be great if we could set the id manually in the property panel.

**Comment by Khalid-Nowaf** [comment]
I would second this one. This will help us to animate things easily. it would also a be good if there was an ID for each group.

**Comment by aewshopping** [comment]
Also looking for this solution and reached this thread! + 1 from me too...

**Comment by fplusf** [comment]
would be cool to have this feature supported from the library itself, but here I found an approach that worked for me, here you can assign whatever id you want, maybe even depending on the type of shape etc.

```
excalidrawElements.forEach(async (element) => {
                const convertedSVG = await exportToSvg({
                  elements: [element],
                  appState: {
                    ...appState,
                    exportBackground: false
                  },
                  exportPadding: -3,
                });

                convertedSVG.setAttribute('id', element.id.toString());

                setSvg([...(svg ? svg : []), convertedSVG]);
              });
```
